Gruntfile.js

```diff
@@ -41,6 +41,22 @@ module.exports = function(grunt) {
             mixinPrefix: ''
           }
         }
+      },
+
+      // create a helper css file for BiDi-proof icons
+      bidi: {
```

Comment:
I would just make and maintain the file by hand. There aren't going to be that many bdi icons.